### DISCUSSION TITLE
Chem - Disabling on objects.

### DISCUSSION THREAD

**Discussion by Tetlys** [Vdauphin/HeartsAndMinds#1494] [Question and answer]
Hey there!

I was wondering if there is a variable I could assign to certain units to prevent them from becoming contaminated? I'm using AI models for the logic box, and they continually get contaminated, then people spawn and take damage from the start. 

Is there any code I could add, or that already exists, to prevent propagation to particular objects?

Thanks!

**Comment by Vdauphin** [comment]
Hello,

> Is there any code I could add, or that already exists, to prevent propagation to particular objects?

No

> I'm using AI models for the logic box,

May be use `agent`, I don't think they will be contaminated as there are not part of `allUnits`

Cheers
